### DISCUSSION TITLE
User settings in Plugin

### DISCUSSION THREAD

**Discussion by SergeoLacruz** [inventree/InvenTree#4605] [Q&A] [answered]
Hi,
Is there a way to define a user setting inside a plugin. I tried set_setting(key, value, user) but this dies not seem to work.
Actually all plugin settings are visible for admins only and there is just one value for all. 
I need a setting that is different for each user (proxy setting). Thanks in advance,

Michael

**Comment by SchrodingersGat** [comment, marked as answer]
Hey Michael, currently the plugin settings are all global, not specific to a particular user. Could look to implement this but no guarantee it will be quick.

**Comment by SergeoLacruz** [comment]
Thanks for the info. I will use the global settings so far.
